signac/common/config.py

```diff
@@ -97,10 +97,18 @@ def _locate_config_dir(search_path):
             search_path = up
 
 
+class _Config(ConfigObj):
```

Comment:
Changes here:
- renamed `Config` to `_Config`
- removed the arbitrary `*args, **kwargs` to limit the API to only the argument we use (`preserve_errors`)
- moved the class definition above where it is used in `_read_config_file` and `_load_config`

Reply:
'encoding' is a kwarg for the parent constructor, and from looking at the source it doesn't look like setting the class member variable does what we want because each instance has the attribute overridden on construction anyway. At this point I think removing is fine since that means that we haven't been specifying the encoding historically anyway.

Reply:
Great, that was the same thing I saw with the parent constructor not using the class member.